### PR TITLE
Init monorepo

### LINKED ISSUE #1547: Adjust components location and imports in prep for sharing
```
        /components
            /src
        /manager
            /src
        /docs
            /src
```

### PR DESCRIPTION
**Pull request by mparke**
closes #1547 

Creates
```
/linode-components
/linode-styleguide
```
and adds them as packages in `package.json`

Updates import paths as needed

### CODE REVIEW

**Comment by eatonphil** [comment]
Can we call these directories just `components` and `styleguide`? Having the `linode-` prefix is less cool except for when we actually need the package name.

**Comment by mparke** [comment]
@eatonphil we could, I started that way, but when we eventually want the package names like you said, may as well now?

File: scss/linodes/components/StatusDropdown.scss [hunk 1]
```diff
@@ -34,12 +34,12 @@
...
-        right: -2px;
-        left: initial;
-        margin-top: 0;
-    }
+    //&-menu {
+    //    @extend .dropdown-menu;
+    //    right: -2px;
+    //    left: initial;
+    //    margin-top: 0;
+    //}
```
Comment (eatonphil) [inline scss/linodes/components/StatusDropdown.scss]: Here

File: scss/components/region.scss [hunk 2]
```diff
@@ -63,12 +63,12 @@
...
-        border-radius: 0;
-        min-width: $width;
-        left: -2px;
-    }
+    //.dropdown-menu {
+    //    border: $border;
+    //    border-radius: 0;
+    //    min-width: $width;
+    //    left: -2px;
+    //}
```
Comment (eatonphil) [inline scss/components/region.scss]: Here

File: src/components/Dropdown.js [hunk 3]
```diff
@@ -35,6 +35,9 @@ export default class Dropdown extends Component {
 
     const orientation = this.props.leftOriented === false ? 'dropdown-menu-right' : '';
 
+    // ${this.state.open ? 'open' : ''}
+    // onBlur={this.close}
+    // onClick={this.open}
```
Comment (eatonphil) [inline src/components/Dropdown.js]: What's this?

File: styleguide/src/sections/Lists.js [hunk 4]
```diff
@@ -47,13 +47,13 @@ export default function Lists() {
...
+                  { dataKey: 'datacenter', label: 'Datacenter' },
                   { cellComponent: ButtonCell, text: 'Edit' },
                 ]}
                 data={[
-                  { label: 'linode-1', ip_address: '192.168.0.0', region: 'Huston, TX' },
-                  { label: 'linode-2', ip_address: '192.168.0.0', region: 'Huston, TX' },
-                  { label: 'linode-3', ip_address: '192.168.0.0', region: 'Huston, TX' },
+                  { label: 'linode-1', ip_address: '192.168.0.0', datacenter: 'Huston, TX' },
+                  { label: 'linode-2', ip_address: '192.168.0.0', datacenter: 'Huston, TX' },
+                  { label: 'linode-3', ip_address: '192.168.0.0', datacenter: 'Huston, TX' },
```
Comment (eatonphil) [inline styleguide/src/sections/Lists.js]: Why are these datacenter now? did I break them in my region -> datacenter prs?

File: src/components/Dropdown.js (hunk 3 above)
Comment (mparke) [inline src/components/Dropdown.js]: weird. removing.

**Pull request merged**